src/domain/dto/deleteVirtualHost.go

```diff
@@ -0,0 +1,24 @@
+package dto
+
+import "github.com/goinfinite/os/src/domain/valueObject"
+
+type DeleteVirtualHost struct {
+	PrimaryVirtualHost valueObject.Fqdn      `json:"primaryVirtualHost"`
+	Hostname           valueObject.Fqdn      `json:"hostname"`
```

Comment:
Actually, the `PrimaryVirtualHost` serves as the injection of the primary vhost itself into the UC to compare the hostname to be removed with the primary vhost, precisely to prevent users from removing the primary vhost.  

To avoid misunderstandings, I moved this property below the `Hostname`, and removed its JSON tag, similar to what was done with the operators.